SvalinnPlugin.cpp

```diff
@@ -2,7 +2,7 @@
 #include <sstream>
 
 #include "CubitMessageHandler.hpp"
-#include "CubitInterface.hpp"
+#include "CubitCoreformInterface.hpp"
```

Comment:
What if this was something like:
```
#ifdef CUBIT_VERSION < XX.YY
#include "CubitInterface.hpp"
#define MESSAGE_HANDLER CubitInterface::get_cubit_message_handler()
#else
#include "CubitCoreformInterface.hpp"
#define MESSAGE_HANDLER CubitMessage::get_message_handler()
#endif
```
and below is something like:

``` 
CubitMessageHandler* console = MESSAGE_HANDLER;
```

Reply:
I did something similar

but in a header file so it doesn't add to much copy paste...